docsets/Entitas/docset.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Entitas",
+    "version": "0.34.0",
+    "archive": "Entitas.tgz",
+    "author": {
+        "name": "Simon Schmid",
+        "link": "https://twitter.com/s_schmid"
+    },
+    "aliases": ["Entitas",
+                "Entitas-CSharp",
+                "ECS",
+                "Entity",
+                "Component",
+                "System",
+                "Framework"
+              ],
+
+    "specific_versions": [
+        {
```

Comment:
You added the docset to `docsets/Entitas/versions/0.34.0/Entitas.tgz`. If you want specific versions support you need to add it to the JSON too.